### DISCUSSION TITLE
Wezterm integration is not fully working with WSL2

### DISCUSSION THREAD

**Discussion by Penguin-jpg** [mrjones2014/smart-splits.nvim#221] [Q&A]
I can navigate from wezterm to WSL2, but cannot navigate from WSL2 to wezterm.
I saw there was a pull request that fixed this, so I don't know why I still encounter this problem :(.
I also checked the wezterm_cli_path in the config and it was "wezterm.exe" so it should be correct.
I am using v0.10.0 nvim, v1.5.0 smart-splits.nvim, the latest version of wezterm (nightly), and ubuntu 22.04 for wsl2.
Anyone encounter the same problem?

**Comment by tdzienniak** [comment]
It works, you just need to call the `setup` function like this (I'm using `lazy`) so the wezterm CLI path is set correctly to `wezterm.exe` in WSL:
```lua
return {
    'mrjones2014/smart-splits.nvim',
    config = function()
        require('smart-splits').setup({})
    end
}
```

**Reply by Penguin-jpg** [reply]
Thanks for replying! I did call setup, but it still doesn't work. I tried to remove the VeryLazy event, but it became that I can only move between wezterm and nvim, not within nvim.

Here is my config:
nvim:
![image](https://github.com/user-attachments/assets/e4986e75-76ee-41c2-b0f7-6dc0bcb6b649)
![image](https://github.com/user-attachments/assets/8746b75b-b187-40bd-aadc-445d72fdfe96)

wezterm:
![image](https://github.com/user-attachments/assets/fdc40c74-5da7-4bf6-a39a-c80d9ac18f2c)
![image](https://github.com/user-attachments/assets/002e61e9-0485-46ea-a5fb-0c1c310d8581)

**Reply by gaving** [reply]
Do you definitely have `wezterm.exe` on your `PATH`? 

This is the commit that made the change:- https://github.com/mrjones2014/smart-splits.nvim/commit/6e4190351f4c86569bc46ad41badca95d3ec5803

Just setting this up and when I put `/mnt/c/software/wezterm/` in my PATH things worked alright.

**Reply by Penguin-jpg** [reply]
I tried it, but it still didn't work. Maybe it has something to do with my environment.
The tmux integration works great so I will stick to it now. Thanks for the help!

**Comment by jayhathaway** [comment]
I have the same issue. Anyone have an idea?

**Reply by mrjones2014** [reply]
Is `wezterm` on your `$PATH`? Can you check if this PR helps? https://github.com/mrjones2014/smart-splits.nvim/pull/267

**Reply by jayhathaway** [reply]
Yes wezterm.exe is in my wsl path. I am also using the lazy.vim starter package so I'm wondering if something there is conflicting with the commands.
